### DISCUSSION TITLE
How to export unique glyph names?

### DISCUSSION THREAD

**Discussion by q962** [ryanoasis/nerd-fonts#1336] [Q&A] [answered]
glyph  names be repeated

I want to use freetype to export glyph name.
As a result, I found a lot of duplicate names.

I found some abbreviations in the class names:
- cod
- oct
- fa
- md

How can I get the full name?

If it's not possible, I can only get what I need by parsing the https://www.nerdfonts.com/assets/css/webfont.css file. :<

**Comment by Finii** [comment, marked as answer]
Not sure what you need, but https://github.com/ryanoasis/nerd-fonts/blob/master/glyphnames.json is not useful?

<img width="682" alt="Screenshot 2023-08-08 at 10 39 09" src="https://github.com/ryanoasis/nerd-fonts/assets/16012374/bf9d5251-e08b-4cbb-a0c5-3d0ab59a3975">

**Reply by q962** [reply]
it's great!
